aikau/src/main/resources/alfresco/layout/FixedHeaderFooter.js

```diff
@@ -234,12 +235,64 @@ define(["alfresco/core/ProcessWidgets",
             }
          ]);
 
+         // Add resize listeners to the header/footer
+         setTimeout(lang.hitch(this, this.addHeaderResizeListener));
```

Comment:
It might be useful to put a comment here to explain what setTimeout does without the delay. I had to look it up because it was new to me :)